config/initializers/cookie_rotation.rb

```diff
@@ -0,0 +1,36 @@
+# See `docs/cookie_rotation.md` for details.
+#
+# This initializer can be kept for future
+# rotation or it can be deleted.
+#
+# It will only take action if there is an
+# `old_secret_key_base` secret in `config/secrets.yml`.
+#
+# NOTE: any changes to rails `action_dispatch` encryption config will
+# impact this functionality, rendering it in need of revisiting - manual
+# testing.
+#
+return unless Rails.application.secrets.old_secret_key_base.present?
+
+Rails.application.configure do
+  # Not technically necessary, as this is the current default,
+  # but it is to line up with the config below.
+  config.action_dispatch.use_authenticated_cookie_encryption = false
+
+  # From:
+  # https://www.gitmemory.com/issue/rails/rails/39964/668147345
```

Comment:
This is a link to details of an issue with this approach, where an extra parameter `value` is added to the cookie. The original article at https://blog.kmkonline.co.id/rotating-ruby-on-rails-secret-key-base-9a4dcdf0d817 suggests having `before_action { session.delete :value }` in the base controller as a work-around. Do we need this?

Reply:
I just linked to this because it refers to the origin artcile but handles our specific issue which is that we are still using the older cipher (we should move to the new cipher at some point). I have not encountered the `value` issue it refers to

Reply:
The original article solution does not work because it does not include the `signed_secret` element, in particular, amongst other problems